monster-cards/src/widgets/about/createAboutPage.ts

```diff
@@ -13,7 +14,7 @@ const createAboutPage = createWidgetBase
 			childNodeRenderers: [
 				function(this: AboutPage): DNode[] {
 					const heading = d('h1', { innerHTML: 'About' });
-					const jumbotron = d('div.jumbotron', {}, [ heading ]);
+					const jumbotron = d(`div.${ css.jumbotron.replace(/\s/g, '.') }`, {}, [ heading ]);
```

Comment:
I thought we were going to write a reducer to return the correct class structure for the options?